### DISCUSSION TITLE
Move examples from file menu to stand on its own

### DISCUSSION THREAD

**Discussion by jonathan-s** [processing/p5.js-web-editor#2928] [General]
#### Nature of issue?
- Existing feature enhancement

#### Feature enhancement details:
Right now examples are quite hidden in the file menu. For a first time user you'd want this to be front and center so the user can get an impression (and possibly inspiration) of what can be done. 

So I'd argue that examples should be moved to the main navigation for visibility. I was looking for a while to see if there was any examples. The first section I was looking at was help and feedback. So if we don't want it to be part of the main navigation, they should at least be moved there in my opinion.

**Comment by siddhant1** [comment]
Maybe a separate section in the header besides help & feedback?

**Comment by catarak** [comment]
thanks for reporting! i think i used the Processing application to decide that "Examples" should be under "File", but i can see how that could be too buried. since all of the other menu items are dropdowns, could it be weird that "Examples" would be the only one that's not a dropdown?

**Comment by jonathan-s** [comment]
Personally I wouldn’t find it weird if it was the only thing that didn’t
have a Dropdown

On Thu 21. Mar 2019 at 21:28, Cassie Tarakajian <notifications@github.com>
wrote:

> thanks for reporting! i think i used the Processing application to decide
> that "Examples" should be under "File", but i can see how that could be too
> buried. since all of the other menu items are dropdowns, could it be weird
> that "Examples" would be the only one that's not a dropdown?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/processing/p5.js-web-editor/issues/959#issuecomment-475389812>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACBsEp49NH_8-Q8F40AmT0nbYcHTNWb4ks5vY-t7gaJpZM4b--ix>
> .
>
-- 
Sent from Gmail Mobile

**Comment by catarak** [comment]
another concern is too many nav items. i don't feel like i have the UI/UX expertise to answer these questions.

**Comment by bmoren** [comment]
I'd maybe say that they should be moved to the help & feedback menu, since they are more 'helpful resources' than file functions.

breaking it out to it's own seems fine too!
